backend/inspirehep/records/marshmallow/literature/bibtex.py

```diff
@@ -234,7 +235,8 @@ def get_title(self, data):
         if "subtitle" in title_dict:
             title_parts.append(title_dict["subtitle"])
         return ": ".join(
-            f"{{{latex_encode(part, contains_math=True)}}}" for part in title_parts
+            f"{{{latex_encode(remove_tags(remove_tags(part, allowed_tags='DUMMYROOTTAG')), contains_math=True)}}}"
```

Comment:
can we do it in a loop? it's getting a hard to read having everything in the formatted string